### PR TITLE
update branding

### LINKED ISSUE #636: Remove "Data to Everything" branding.
Hey Team, 

Just a small one here, we need to remove the reference to "Data to Everything" Platform. 

Just doing my part to make brand happy :)

https://github.com/signalfx/splunk-otel-collector-chart/blob/921de2890bde494aeb52756036dcb1cc11fcc5e2/helm-charts/splunk-otel-collector/values.yaml#L20

Can update to "The Unified Security and Observability Platform" as per splunk.com

### PR DESCRIPTION
**Pull request by atoulme**
Fixes #636

### CODE REVIEW

File: helm-charts/splunk-otel-collector/values.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ fullnameOverride: ""
 clusterName: ""
 
 ################################################################################
-# Splunk Data-to-Everything Platform configuration.
+# Splunk The Unified Security and Observability Platform configuration.
```
Comment (dmitryax) [inline helm-charts/splunk-otel-collector/values.yaml:20]: This is a configuration of a destination that needs to be differentiated from Splunk Observability. If we include Observability word here, it becomes pretty confusing given that we have another destination called "Splunk Observability". cc @matthewmodestino @aunshc

Comment (matthewmodestino) [reply helm-charts/splunk-otel-collector/values.yaml:20]: I agree, I would just go with Splunk Cloud or Splunk Enterprise. We can leave the marketing to the marketers ;)

Comment (dmitryax) [reply helm-charts/splunk-otel-collector/values.yaml:20]: "Splunk Cloud / Splunk Enterprise configuration" sounds good to me 👍

Comment (mtwo) [reply helm-charts/splunk-otel-collector/values.yaml:20]: +1, sounds good

Comment (atoulme) [reply helm-charts/splunk-otel-collector/values.yaml:20]:
```suggestion
# Splunk Cloud / Splunk Enterprise configuration.
```

File: helm-charts/splunk-otel-collector/values.schema.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
       "minLength": 1
     },
     "splunkPlatform": {
-      "description": "Splunk Data-to-Everything Platform configuration",
+      "description": "Splunk The Unified Security and Observability Platform configuration",
```
Comment (atoulme) [inline helm-charts/splunk-otel-collector/values.schema.json:29]:
```suggestion
      "description": "Splunk Cloud / Splunk Enterprise configuration",
```

File: ci_scripts/sck_otel_values.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# Splunk Data-to-Everything Platform configuration.
+# Splunk The Unified Security and Observability Platform configuration.
```
Comment (atoulme) [inline ci_scripts/sck_otel_values.yaml:5]:
```suggestion
# Splunk Cloud / Splunk Enterprise configuration.
```

**Review by dmitryax — APPROVED** [review]

**Pull request merged**